api/v1beta1/foundationdbcluster_types.go

```diff
@@ -484,12 +484,18 @@ func (processGroupStatus *ProcessGroupStatus) IsExcluded(remainingMap map[string
 // NewProcessGroupStatus returns a new GroupStatus for the given processGroupID and processClass.
 func NewProcessGroupStatus(processGroupID string, processClass ProcessClass, addresses []string) *ProcessGroupStatus {
 	return &ProcessGroupStatus{
-		ProcessGroupID:         processGroupID,
-		ProcessClass:           processClass,
-		Addresses:              addresses,
-		Remove:                 false,
-		Excluded:               false,
-		ProcessGroupConditions: make([]*ProcessGroupCondition, 0),
+		ProcessGroupID: processGroupID,
+		ProcessClass:   processClass,
+		Addresses:      addresses,
+		Remove:         false,
+		Excluded:       false,
+		ProcessGroupConditions: []*ProcessGroupCondition{
+			NewProcessGroupCondition(MissingProcesses),
```

Comment:
I also thought about adding a new `Initialized` condition that will be reset the first time it appears in the update status reconciler.